File: IsometricShooter/Scripts/JS/GUI/AmmoGui.js
#pragma strict

var playerWeaponList : WeaponListController; //Reference to all weapons of player
var ammo : UI.Text; //GUI text to show ammo of player

private var selectedWeapon : WeaponConf; //currently selected weapon

function OnGUI () {

	selectedWeapon = playerWeaponList.GetWeapon(); //set selected weapon
	//ammo.text = selectedWeapon.title +" : " + selectedWeapon.currentAmmunition + " / " + selectedWeapon.maximumAmmunition;
	ammo.text = "" + selectedWeapon.currentAmmunition;

}

File: IsometricShooter/Scripts/CS/GUI/AmmoGui.cs
using UnityEngine;
using UnityEngine.UI;
using System.Collections;

public class AmmoGui : MonoBehaviour {

	public WeaponListController playerWeaponList; //Reference to all weapons of player
	public Text ammo; //GUI text to show ammo of player
	
	private WeaponConf selectedWeapon; //currently selected weapon

	void OnGUI () {

		selectedWeapon = playerWeaponList.GetWeapon(); //set selected weapon
		//ammo.text = selectedWeapon.title +" : " + selectedWeapon.currentAmmunition + " / " + selectedWeapon.maximumAmmunition;
		ammo.text = "" + selectedWeapon.currentAmmunition;

	}
	
}
